File: src/employeemanager/util/EmployeePrinter.java
package employeemanager.util;

import employeemanager.model.Employee;
import employeemanager.model.Manager;
import employeemanager.model.PartTimeEmployee;

public class EmployeePrinter {

    public static void print(Employee employee) {
        System.out.println("[Employee] " + employee.getFullName() + 
        		" pay=" + employee.getPay());
    }

    public static void print(Manager manager) {
        System.out.println("[Manager] " + manager.getFullName() +
                " teamSize=" + manager.getTeamSize() +
                " pay=" + manager.getPay());
    }

    public static void print(PartTimeEmployee parttime) {
        System.out.println("[PartTime] " + parttime.getFullName() +
                " weeklyPay=" + parttime.getPay());
    }
}

File: Part 3/EmployeeManager/utils/EmployeePrinter.cs
using System;
using EmployeeManager.Models;

namespace EmployeeManager.util
{
    public static class EmployeePrinter
    {
        public static void Print(Employee employee)
        {
            Console.WriteLine($"[Employee] {employee.FullName} pay= {employee.GetPay()}");
        }

        public static void Print(Manager manager)
        {
            Console.WriteLine(
                $"[Manager] {manager.FullName} Team Size= {manager.Team.Count} Pay= {manager.GetPay()}"
            );
        }

        // Fully-qualified type name to avoid any namespace confusion
        public static void Print(EmployeeManager.Models.PartTimeEmployee partTime)
        {
            Console.WriteLine($"[PartTime] {partTime.FullName} Weekly pay= {partTime.GetPay()}");
        }
    }
}
